plugins/plugin-docker/che-plugin-openshift-client/pom.xml

```diff
@@ -36,6 +36,11 @@
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>3.4.1</version>
```

Comment:
Guys. I know it's your branch. But could you follow the general rule to not to declare versions in maven submodules. It can save us some time in a future.